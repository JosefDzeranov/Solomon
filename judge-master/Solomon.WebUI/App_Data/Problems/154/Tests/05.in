5
0 10 100
10 20 80
20 30 60
30 40 40
40 50 10



