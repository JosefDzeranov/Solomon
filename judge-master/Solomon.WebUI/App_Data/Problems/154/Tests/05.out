0 10 10
10 20 20
20 30 30
30 40 40
40 50 50
50 40 70
70 30 80
80 20 90
90 10 100
